### DISCUSSION TITLE
モデルのクレジット表示について

### DISCUSSION THREAD

**Discussion by vdyakuin** [pixiv/vroid-sdk-developers#101] [Q&A]
お世話になっております

「モデルのクレジット表示」について、モデル名と作者名が「モデルデータの利用条件確認画面」と「VRMmeta」で格納されている値が違う場合があります。
どちらの値を表示すればよろしいでしょうか？

↓例
![image](https://github.com/pixiv/vroid-sdk-developers/assets/98076283/637819cf-2d1f-4b30-b36f-c1bdf3751a8f)
![image](https://github.com/pixiv/vroid-sdk-developers/assets/98076283/e39123eb-7203-481c-af91-f6517e949d6d)

お忙しい中恐れ入りますがご回答よろしくお願いいたします。

**Comment by nyamadan** [comment]
大変お待たせして申し訳ございません。こちら、調査いたします。

**Reply by vdyakuin** [reply]
お忙しい中ご対応ありがとうございます。よろしくお願いいたします。
